examples/riot-kitchen-sink/src/index.js

```diff
@@ -0,0 +1,6 @@
+import riot from 'riot';
+// eslint-disable-next-line
```

Comment:
Why disable eslint? if you're going to disable something, only disable specific rules.